src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.html

```diff
@@ -1,45 +1,17 @@
 <ng-template #usageTooltipTpl>
-  <table *ngIf="!showMultisiteTooltip">
-    <tr>
-      <td class="text-left me-1">Used:</td>
-      <td class="text-right"><strong> {{ isBinary ? (used | dimlessBinary) : (used | dimless) }}</strong></td>
-    </tr>
-    <tr *ngIf="calculatePerc && showFreeToolTip">
-      <td class="text-left me-1">Free:</td>
-      <td class="'text-right"><strong>{{ isBinary ? (total - used | dimlessBinary) : (total - used | dimless) }}</strong></td>
-    </tr>
-    <tr *ngIf="customLegend">
-      <td class="text-left me-1">{{ customLegend }}:</td>
-      <td class="text-right"><strong>{{ isBinary ? (customLegendValue | dimlessBinary) : (customLegend[1] | dimless) }}</strong></td>
-    </tr>
-  </table>
-  <table *ngIf="showMultisiteTooltip">
-    <tr>
-      <td class="text-left">Total Shards:&nbsp;</td>
-      <td class="text-right"><strong> {{ total }}</strong></td>
-    </tr>
-    <tr *ngIf="calculatePerc">
-      <td class="text-left">Transferred Shards:&nbsp;</td>
-      <td class="'text-right"><strong>{{ used }}</strong></td>
-    </tr>
-  </table>
+  <div>
+    <span>
+      {{ isBinary ? (used | dimlessBinary) : (used | dimless) }}
+      <span *ngIf="calculatePerc">/ {{ isBinary ? (total | dimlessBinary) : (total | dimless)}}</span>
+    </span>
+  </div>
```

Comment:
Given that the Carbon meter component is something like the following, why not using that?:
```html
<ibm-meter-chart [data]="data" [options]="options"></ibm-meter-chart>
```
The purpose of the progress bar is not to display data, but to inform the user about the evolution of an ongoing process.

Reply:
yeah I will try to integrate the meter chart and see if it will be useful instead of using progressbar,
Thanks Ernesto

Reply:
Updated!